src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -141,7 +141,7 @@
       end
 
       it do
-        expect(flash[:error]).to eq('Unable to submit: Validation failed: Bs request actions is invalid, ' \
+        expect(flash[:error]).to eq('Unable to submit: Validation failed: Bs request actions sourceupdate is not included in the list, ' \
```

Comment:
With the addition of `accepts_nested_attributes_for :bs_request_actions` in the `BsRequest` model, validations are triggered for the `bs_request_actions` association whenever saving a `BsRequest` instance. This changes the error message of this spec.

See upstream: https://github.com/rails/rails/blob/89d1d39c8f668d2fa0aeecb97431095f3f4e7ebc/activerecord/lib/active_record/nested_attributes.rb#L341